bigtable-hbase-1.x-parent/bigtable-hbase-tools/README.md

```diff
@@ -0,0 +1,104 @@
+#HBase Tools
+
+This folder contains tools to help HBase users with migrations to Cloud Bigtable.
+Pipelines to import and export data are under [bigtable-beam-import](bigtable-dataflow-parent/bigtable-beam-import/README.md).
+
+## Setup 
+
+To use the tools in this folder, you can download them from the maven repository, or
+you can build them using Maven.
+
+[//]: # ({x-version-update-start:bigtable-dataflow-parent:released})
+### Download the jars
+
+todo: get correct link, also should I provide a wget command?
```

Comment:
We need to release this tool to get a link from Maven. I will get it in a day or 2. We can leave the link out during that time.